### DISCUSSION TITLE
What's inside of start.sh.

### DISCUSSION THREAD

**Discussion by GG-GhostGaming** [RikkaApps/Shizuku#301] [Q&A]
What is the exact code that is inside of the start script? 

I want to know for a project that I'm working on to automatically start shizuku at boot. I will eventually try just pulling the file from ADB itself on my computer, but I don't feel like installing platform tools again. (Had to reimage my computer recently and no longer have it installed lol)

**Comment by e-t-l** [comment]
Here's the source code for the script. You're on github friend, you're allowed to look through the code yourself ;) 

https://github.com/RikkaApps/Shizuku/blob/5f282e98209f09100aa6f2fa0e813b15dcec67cc/manager/src/main/res/raw/start.sh

**Reply by GG-GhostGaming** [reply]
Thanks!.
